app/src/ui/app.tsx

```diff
@@ -352,6 +352,12 @@ export class App extends React.Component<IAppProps, IAppState> {
     }
 
     this.checkIfThankYouIsInOrder()
+
+    if (isWindowsAndNoLongerSupportedByElectron()) {
+      this.props.dispatcher.setBanner({
+        type: BannerType.WindowsVersionNoLongerSupported,
+      })
+    }
```

Comment:
I just noticed… won't this be shown every time the app starts? Should we show it only the first time maybe? 🤔

Reply:
Maybe? Having it show on each launch was intentional, we want to signal that users running on these operating systems aren't going to be receiving updates so some level of nagging is prudent IMO. We could reduce it to only show once every so often but that would mean adding some more logic to this. What interval do you think is reasonable?

Reply:
I updated this to be once every 90 days, feels like a decent compromise right?